src/App.jsx

```diff
@@ -1,34 +1,26 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { useState, useEffect } from 'react'
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { getArticles } from '../api';
+import { ArticlesPage } from './pages/ArticlesPage';
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([]);
```

Comment:
According to your plan this shouldn't be here. Due to the size of this project, this absolutely shouldn't be here.